uefi-services/Cargo.toml

```diff
@@ -3,16 +3,12 @@ name = "uefi-services"
 version = "0.15.0"
 authors = ["Gabriel Majeri <gabriel.majeri6@gmail.com>"]
 edition = "2021"
-description = "Higher-level utilities for uefi-rs"
+description = "Higher-level utilities for the uefi-rs crate."
 repository = "https://github.com/rust-osdev/uefi-rs"
 keywords = ["uefi", "efi"]
 categories = ["embedded", "no-std", "api-bindings"]
 license = "MPL-2.0"
-
-[badges]
-travis-ci = { repository = "rust-osdev/uefi-rs" }
-is-it-maintained-issue-resolution = { repository = "rust-osdev/uefi-rs" }
-is-it-maintained-open-issues = { repository = "rust-osdev/uefi-rs" }
+readme = "README.md"
```

Comment:
This doesn't need to be set explicitly, the default will pick up `README.md`: https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field

Reply:
The official doc is wrong. I experienced multiple cases where crates.io wouldn't show up the Readme without this key. There are some crates out there where I contributed this change and after the next release, the README did show up on crates.io. 
![Uploading Screenshot_2022-11-17-22-00-14-78_40deb401b9ffe8e1df2f1cc5ba480b12.jpg…]()

Reply:
We have it for the main crate. We should at least stay consistent.. But.. Better be safe?

Reply:
> There are some crates out there where I contributed this change and after the next release, the README did show up on crates.io.

Wonder if it could be this? https://github.com/rust-lang/crates.io/issues/3941#issuecomment-922706038

At any rate, I'm fine with keeping this field then, better safe than sorry :)